extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -41,6 +41,22 @@ tags: {  # <amp-story>
       protocol: "https"
     }
   }
+  attrs: {
+    name: "parallax-fx",
+    value: ""
+  }
+  attrs: {
```

Comment:
fyi these could be made into an attribute list to be shared between tags
https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L184